metrics/prometheus_machine.go

```diff
@@ -192,7 +193,29 @@ func NewPrometheusMachineCollector(i infoProvider, includedMetrics container.Met
 			},
 		}...)
 	}
+	if includedMetrics.Has(container.VmStatMetrics) {
```

Comment:
please add unit tests for prometheus metrics, see test in https://github.com/google/cadvisor/tree/master/metrics